### PR TITLE
CORTX-34368: Fix for the cortx-re codacy issues to test dashboard

### PR DESCRIPTION
**Pull request by pankajgborole**
# Problem Statement
Fix of minor codacy issues, in order to validate the Dashboard issue count for codacy for cortx-re repo
Validate the same by running dependent jobs 

# Design
Codacy has reported few issues on cortx-re repo. choose minor issues and fix them in code which will not have impact on functionality. 

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [x] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] Jira ID/ COMMUNITY is prefixed in PR title
      Prefix Syntax - 
        `CORTX-<5 digit JIRA ID>: <PR Title>` - for Seagate Employees
        `COMMUNITY: <PR Title>`  - for open source contributors
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: jenkins/internal-ci/generic/branch/cortx-py-utils.groovy [hunk 1]
```diff
@@ -89,7 +89,7 @@ pipeline {
                     def releaseBuild = build job: 'Release', propagate: true
                      env.release_build = releaseBuild.number
                     env.release_build_location = "http://cortx-storage.colo.seagate.com/releases/cortx/github/$branch/$os_version/${env.release_build}"
-                    env.cortx_images = releaseBuild.buildVariables.cortx_all_image +"\n" +releaseBuild.buildVariables.cortx_rgw_image +"\n" +releaseBuild.buildVariables.cortx_data_image +"\n" +releaseBuild.buildVariables.cortx_control_image
```
Comment (shailesh-vaidya) [inline jenkins/internal-ci/generic/branch/cortx-py-utils.groovy:92]: Please validate that cortx-py-utils package creation is working with changes

File: jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy [hunk 2]
```diff
@@ -27,7 +27,7 @@ pipeline {
             name: 'CORTX_IMAGE'
         )
         choice (
-            choices: ['sanity','regression','audit_log','cli_framework','cmd_framework','conf_store','consul_service','iem_framework','kv_store','message_bus','support_bundle','validator'],
```
Comment (shailesh-vaidya) [inline jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy:30]: Please validate that cortx-py-utils PR pipeline is working with changes

File: jenkins/automation/alex.groovy [hunk 3]
```diff
@@ -122,7 +122,7 @@ pipeline {
               script {
                 echo "Get the scanned alex file."
                 sh "ls *.alex > listAlexFiles"
-                def files = readFile( "listAlexFiles" ).split( "\\r?\\n" );
```
Comment (shailesh-vaidya) [inline jenkins/automation/alex.groovy:125]: Please validate that alex report generation is working fine with changes

**Review by shailesh-vaidya — COMMENTED** [review]
- Please update all fields in the PR checklist template 
- Please add the Jenkins pipeline used for validating changes.

**Review by gauravchaudhari02 — APPROVED** [review]

File: jenkins/internal-ci/generic/branch/cortx-py-utils.groovy (hunk 1 above)
Comment (pankajgborole) [reply jenkins/internal-ci/generic/branch/cortx-py-utils.groovy:92]: 1) Checked and updated the applicable fields in PR Checklist template 

2) py-util is working fine
https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/Generic/job/py-utils/742/

2) Validation on cortx-py-utils is in progress once its done will update Jenkins pipeline details

File: jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy (hunk 2 above)
Comment (pankajgborole) [reply jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy:30]: cortx-py-utils working fine
https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/custom-cortx-py-utils/2947/

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**